src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -206,43 +166,25 @@ public void stopFixedArm() {
         setMotor(ClimbConstants.STOP_SPEED, climbFixedMotor);
     }
 
-    public void extendAngledArm() {
-        setMotor(testSpeedExtend.getValue().getDouble(), climbDynamicMotor);
+    public void extendDynamicArm() {
+        setMotor(HIGH_RUNG_HEIGHT_INCH * ENCODER_TICKS_PER_INCH, climbDynamicMotor);
     }
 
-    public void retractAngledArm() {
-        setMotor(testSpeedRetract.getValue().getDouble(), climbDynamicMotor);
+    public void retractDynamicArm() {
+        setMotor(RETRACT_HEIGHT_INCH * ENCODER_TICKS_PER_INCH, climbDynamicMotor);
+    }
+
+    public void retractDynamicArmFully() {
+        setMotor(2 * ENCODER_TICKS_PER_INCH, climbDynamicMotor);
     }
 
     public void stopAngledArm() {
         setMotor(ClimbConstants.STOP_SPEED, climbDynamicMotor);
```

Comment:
`setMotor()`'s `value` parameter is position in encoder ticks, not percent output.